python/processors/nft_orderbooks/models/nft_marketplace_bid_models.py

```diff
@@ -35,6 +36,7 @@ class CurrentNFTMarketplaceBid(Base):
 
 class NFTMarketplaceBid(Base):
     __tablename__ = "nft_marketplace_bids"
+    __table_args__ = {"schema": "nft_marketplace"}
```

Comment:
can we move this to a shared constant or something to avoid repeating it? I think you can probably also add it to the `Base`

Reply:
@rtso fair optimization.